db/seeds/profile_link_sites.yml

```diff
@@ -138,3 +138,33 @@ trakt:
   name: Trakt
   validate_find: '\A(https?://)?(www.)?(trakt.tv/)?(users/)?(?<username>.+)\z'
   validate_replace: 'https://trakt.tv/users/\k<username>'
+myanimelist:
+  id: 29
+  name: MyAnimeList
+  validate_find: '\A(https?://)?(www.)?(myanimelist.net/)?(profile/)?(?<username>[a-zA-Z0-9_\-]+)\z'
+  validate_replace: 'https://myanimelist.net/profile/\k<username>'
+animeplanet:
+  id: 30
+  name: AnimePlanet
+  validate_find: '\A(https?://)?(www.)?(anime-planet.com/)?(users/)?(?<username>[a-zA-Z0-9_\-]+)\z'
+  validate_replace: 'https://anime-planet.com\users\k<username>'
```

Comment:
Honestly I don't really like the thought of giving free advertising to AnimePlanet, AniDB, AniList, or MAL (not that MAL needs it from us lmao)